app/controllers/posts_controller.rb

```diff
@@ -2,8 +2,10 @@ class PostsController < ApplicationController
   def index
     if params[:user_id].present?
       # List posts of a specific user
+
       @user = User.find(params[:user_id])
-      @posts = @user.posts
+      # @posts = @user.posts
```

Comment:
- [ ] Please remove any commented-out code it makes your codebase look unorganised and harder to read.